core/controllers/base_test.py

```diff
@@ -352,6 +359,19 @@ def test_logout_page(self):
             datetime.datetime.utcnow() > datetime.datetime.strptime(
                 expiry_date[1], '%a, %d %b %Y %H:%M:%S GMT',))
 
+    def test_logout_page_redirect_not_dev_mode(self):
```

Comment:
I think you can write a better name for this test method following this pattern:
`test_action_withCondition_withAnotherCondition_hasExpectedOutcome` (from [wiki guidelines](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests))